demisto_sdk/commands/unify_packs/packs_unifier.py

```diff
@@ -0,0 +1,191 @@
+import logging
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+
+import click
+from demisto_sdk.commands.common.constants import PACKS_DIR
+from demisto_sdk.commands.common.content.objects.pack_objects.pack import Pack
+from demisto_sdk.commands.common.tools import arg_to_list
+from demisto_sdk.commands.create_artifacts.content_artifacts_creator import (
+    IGNORED_PACKS, ArtifactsManager, ContentObject, create_dirs, delete_dirs,
+    dump_pack, zip_pack_zips, zip_packs)
+
+EX_SUCCESS = 0
+EX_FAIL = 1
+
+
+class PacksUnifier:
+
+    def __init__(self, pack_paths: str, output: str, content_version: str, zip_all: bool, quite_mode: bool = False):
+        default_args = dict(packs=True, zip=True, cpus=os.cpu_count(), suffix='')
+        self.artifacts_manager = PacksManager(
+            pack_paths=pack_paths,
+            artifacts_path=output,
+            content_version=content_version,
+            all_in_one_zip=zip_all,
+            quite_mode=quite_mode,
+            **default_args,
+        )
+
+    def unify_packs(self):
+        """
+        Compress the given packs to one zip file or zip for each pack (depended on the zip_all flag)
+
+        Returns: tuple (the created zip path or the directory of the pack zips, list of the pack names)
+                 or (None, None) if the pack names are empty
+
+        """
+        if self.artifacts_manager.pack_names:
+            self.artifacts_manager.dump_packs()
+            return self.artifacts_manager.output_path, self.artifacts_manager.pack_names
+
+        else:
+            return None, None
+
+
+class PacksManager(ArtifactsManager):
+    def __init__(self, pack_paths: str, all_in_one_zip: bool, quite_mode: bool, **kwargs):
+        super().__init__(**kwargs)
+        self.init_packs(pack_paths)
+        self.zip_all = all_in_one_zip
+        self.quite_mode = quite_mode
+        self.output_path = f'{self.content_uploadable_zips_path}.zip' if self.zip_all \
+            else str(self.content_uploadable_zips_path)
+
+    def init_packs(self, pack_paths):
+        """
+
+        Args:
+            pack_paths: CSV str with the pack paths
+
+        init dict that map pack name to Pack object and init also the pack_names property
+
+        """
```

Comment:
```suggestion
        """init dict that map pack name to Pack object and init also the pack_names property

        Args:
            pack_paths: CSV str with the pack paths

        """
```

This is the right way to write the docstring, please change the rest as well.
Look at the example here: https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e#file-docstrings-py-L79